README.md

```diff
@@ -79,16 +79,16 @@ See the separate [CONTRIBUTING.md](https://github.com/mozilla/fxa/blob/main/CONT
 
 > This is an example workflow for **fxa**.
 
-After installing **fxa** run `npm start`. Use `./pm2 status` command to check the status of the servers:
+After installing **fxa** run `yarn start`. Use `yarn pm2 status` command to check the status of the servers:
 
-![](http://i.imgur.com/eqL8FiZ.png)
+![](https://i.imgur.com/Kl3qWtb.png)
```

Comment:
This is a pretty tall image, any chance we can retake the screenshot with a smaller text size? 

![Screen Shot 2021-04-12 at 9 51 53 AM](https://user-images.githubusercontent.com/6392049/114405500-bf202a80-9b74-11eb-97a5-39257ba48a49.png)

Reply:
> This is a pretty tall image, any chance we can retake the screenshot with a smaller text size?
> 
> ![Screen Shot 2021-04-12 at 9 51 53 AM](https://user-images.githubusercontent.com/6392049/114405500-bf202a80-9b74-11eb-97a5-39257ba48a49.png)



> This is a pretty tall image, any chance we can retake the screenshot with a smaller text size?
> 
> ![Screen Shot 2021-04-12 at 9 51 53 AM](https://user-images.githubusercontent.com/6392049/114405500-bf202a80-9b74-11eb-97a5-39257ba48a49.png)

working on it!

Reply:
working on it!